homework-1/css/style.css

```diff
@@ -0,0 +1,1220 @@
+  display: flex;
+  flex-direction: column;
+}
+.header .menu__list > li {
+  position: relative;
+  margin: 0 auto;
+}
+.header .menu__link {
+  font-size: 12px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #22BF95;
+}
+.header .header__button-link {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  top: 80px;
+}
+.header .header__button-link a {
+  font-size: 12px;
+  border-radius: 20px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #22BF95;
+  padding: 8px 0;
+  border: 1.5px solid #22BF95;
+  border-radius: 20px;
+}
+.header .header__banner {
+  height: 211px;
+}
+.header .menu__body {
+  position: absolute;
+  top: 64px;
+  left: 0;
+  width: 100%;
+  background-color: #fff;
+  transition: transform 0.5s ease-out;
+  z-index: 1;
+  transform: translateY(-120vh);
+}
+.header .menu__body.active {
+  transform: translateY(0px);
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.header .menu__icon {
+  display: block;
+  width: 24px;
+  height: 18px;
+  position: relative;
+  cursor: pointer;
+}
+.header .menu__icon::before, .header .menu__icon::after, .header .menu__icon span {
+  position: absolute;
+  left: 0;
+  height: 10%;
+  width: 100%;
+  background-color: #fff;
+  transition: all 0.3s ease 0s;
+}
+.header .menu__icon::before, .header .menu__icon::after {
+  content: "";
+}
+.header .menu__icon::before {
+  top: 0;
+}
+.header .menu__icon::after {
+  bottom: 0;
+}
+.header .menu__icon span {
+  top: 50%;
+  transform: scale(1) translate(0px, -50%);
+}
+.header .menu__icon.active span {
+  transform: scale(0) translate(0px, -50%);
+}
+.header .menu__icon.active::before {
+  top: 50%;
+  transform: rotate(-45deg) translate(0px, -50%);
+}
+.header .menu__icon.active::after {
+  bottom: 50%;
+  transform: rotate(45deg) translate(0px, 50%);
+}
+.header .get__link {
+  width: 180px;
+}
+.header .log__link {
+  width: 90px;
+  margin-top: 16px;
+}
+.header__banner {
+  height: 211px;
+}
+.header__banner-title {
+  font-family: "Actor-Regular";
+  font-size: 40px;
+  line-height: 56px;
+  text-align: center;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #fff;
+  padding-top: 30px;
+  padding: 67px 0 14px;
+  max-width: 142px;
+  margin: 0 auto;
+}
+.header__banner-submenu {
+  padding: 0 20px;
+  position: absolute;
+  width: 100%;
+  top: 198px;
+}
+.first-item {
+  padding-right: 26px;
+  padding-top: 6px;
+}
+.submenu__item-middle {
+  padding-top: 5px;
+}
+.last-item {
+  padding-left: 24px;
+  padding-top: 6px;
+}
+.submenu__list {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+  background: #FFFFFF;
+  box-shadow: 0px 12px 24px #F2F2F2;
+  border-radius: 5px;
+  border-top: 2.5px solid #37E63E;
+}
+.submenu__list a {
+  color: #7f7f7f;
```

Comment:
The same, use #7f7f7f(with lowercase) before you declared colors with uppercase - like #7F7F7F. I recommend you to use first one.

Please check it through the file

Reply:
fixed a bug